raiden/tests/integration/transfer/test_mediatedtransfer.py

```diff
@@ -518,7 +532,7 @@ def run_test_mediated_transfer_with_allocated_fee(
     # app1 (provided we've set the fee of the channel)
     with gevent.Timeout(network_wait):
         wait_assert(
```

Comment:
Hummm, what is this test doing? Maybe this is a better tool for the job:

https://github.com/raiden-network/raiden/blob/53dc61c9d86e30cf6f0389e194575c0825bbee4f/raiden/tests/utils/transfer.py#L261

Reply:
Yes, makes sense.